src/forms.css

```diff
@@ -924,7 +924,7 @@ input:checked + .checkbox {
 }
 
 /* state management for switches */
-.switch:hover {
```

Comment:
You'll need to update the switch rules in build-color-variants if you want this to work properly.